packages/circuits/tester/utxo_notehash.test.circom

```diff
@@ -0,0 +1,54 @@
+include "../lib/nullifier.circom";
+include "../lib/note_hash.circom";
+include "../lib/asset_hash.circom";
+include "../lib/spending_pubkey.circom";
+include "../lib/ownership_proof.circom";
+
+template UtxoNoteHash() {
```

Comment:
@barryWhiteHat 
@weijiekoh
This is a testing circuit to check the new nullifier & utxo logic work well.